modules/boot-commons/pom.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>mule-modules</artifactId>
+        <groupId>org.mule.runtime</groupId>
+        <version>4.8.0-SNAPSHOT</version>
+    </parent>
+
+    <groupId>org.mule.runtime.boot</groupId>
+    <artifactId>mule-module-boot-commons</artifactId>
+    <name>Mule Boot Commons</name>
+    <packaging>jar</packaging>
+
+    <properties>
+        <javaModuleName>org.mule.boot.commons</javaModuleName>
+        <formatterConfigPath>../../formatter.xml</formatterConfigPath>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>commons-cli</groupId>
+            <artifactId>commons-cli</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.mule.runtime.boot</groupId>
+            <artifactId>mule-module-boot-api</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+
+        <!-- Needed by the BootModuleLayerValidationBootstrapConfigurer and for creating the container ClassLoader -->
+        <dependency>
+            <groupId>org.mule.runtime.boot</groupId>
+            <artifactId>mule-module-jpms-utils</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+
+        <!-- Needed by the SLF4JBridgeHandlerBootstrapConfigurer, but also to make the logging modules available from the boot layer -->
+        <dependency>
+            <groupId>org.mule.runtime.boot</groupId>
+            <artifactId>mule-module-logging</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.mule.runtime</groupId>
+            <artifactId>api-annotations</artifactId>
+            <version>${muleAnnotationsApiVersion}</version>
+            <scope>provided</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.mule.tests</groupId>
+            <artifactId>mule-tests-unit</artifactId>
+            <version>${project.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.mule.runtime.boot</groupId>
+            <artifactId>mule-module-log4j-boot-configurator</artifactId>
+            <version>4.8.0-SNAPSHOT</version>
```

Comment:
Property and comment like `boot-api`, put it together with the other boot modules.